docker/conf/traefik.toml

```diff
@@ -153,7 +153,6 @@ address = ":8080"
 # Optional
 # Default: ""
 #
-domain = "altis.dev"
```

Comment:
@roborourke I couldn't actually see where this was used. The docs say it's a default, do you know impact of removing this?

Reply:
We could maybe generate that file too. Honestly best off asking @nathanielks but if it all works without it then that’s ok

Reply:
AFAIK this won't impact anything. As you say, it's a default and I'm assuming the value would already be set by the `Host` header.

Reply:
[Docs](https://doc.traefik.io/traefik/v1.7/configuration/backends/docker/) for posterity